File: java/empleados/Empleado.java
public class Empleado {
    
    private String nombre;
    private int edad;
    private double salario;
    public final double PLUS = 300;

    public Empleado(String nombre,int edad){
        this.nombre = nombre;
        this.edad = edad;
        
    }

    public void setSalario(double salario){
        this.salario = salario;
    }

    public String getNombre(){
        return this.nombre;
    }

    public int getEdad(){
        return this.edad;
    }

    public double getSalario(){
        return this.salario;
    }

    @Override
    public String toString(){
        return "Nombre :" + this.nombre + ", edad: \n" + this.edad + ", salario : " + this.salario;
    }


}

File: dart/Empleados/Empleado.dart
class Empleado{
    String _nombre = "";
    int _edad = 0;
    double _salario = 0;
    double PLUS = 200;

  Empleado(nombre,edad,salario){
    this._nombre = nombre;
    this._edad = edad;
    this._salario = salario;
  }

  set setSalario(double salario){
    this._salario = salario;
  }

  @override 
  String toString() {
    return "\nNombre: " + this._nombre +
    "\nEdad: " + this._edad.toString() +
    "\nSalario: " + this._salario.toString();
  }

}